### PR TITLE
pkgbuild file

### PR DESCRIPTION
**Pull request by xeon-zolt**
it is a  pkg build file to build and install it on arch #10

### CODE REVIEW

**Comment by mariobehling** [comment]
Good progress! Please ask your co-developers to review and check what can be done about travis. Cheers.

File: PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Maintainer: xeon-zolt <xeon.harsh@gmail.com>
+pkgname=meilix-systemlock
+pkgver=1
+pkgrel=1
+
+pkgdesc="Melix System Lock"
+arch=('i686' 'x86_64')
```
Comment (meets2tarun) [inline PKGBUILD]: In my opinion,
(i386, amd 64) 
Why did you choose this. Please clarify!

File: PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+url="https://github.com/fossasia/meilix-systemlock"
+license=('GPL')
```
Comment (meets2tarun) [inline PKGBUILD]: GPLv3

File: PKGBUILD (hunk 1 above)
Comment (xeon-zolt) [inline PKGBUILD]: for 32 bit its i686 and for 64 bit its x86_64 in pacman repo

Comment (meets2tarun) [inline PKGBUILD]: But here travis will build in ubuntu environment!

Comment (xeon-zolt) [inline PKGBUILD]: i will add testing for this too in travis its just to create the pacman package
we use travis to test code buddy

**Comment by xeon-zolt** [comment]
https://aur.archlinux.org/packages/meilix-systemlock/

**Comment by niccokunzmann** [comment]
You can merge again for travis.

**Comment by meets2tarun** [comment]
Thank you
#19 close, please

**Pull request merged**